command_line/reflection_viewer.py

```diff
@@ -36,7 +36,7 @@ def __init__(self):
         import libtbx.load_env
 
         # The script usage
-        usage = "usage: %s [options] experiment.json" % libtbx.env.dispatcher_name
+        usage = "usage: %s [options] experiment.expt" % libtbx.env.dispatcher_name
```

Comment:
```suggestion
        usage = "usage: dials.reflection_viewer [options] models.expt"
```

**HOWEVER**, the context suggests that the help string is wrong and it should be `observations.refl` instead of any `.expt`